2Weekly Mission/Index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="Linkbrarystyle.css" rel="stylesheet" />
```

Comment:
일반적인 html, css 파일명은 주로 케밥 케이스를 많이 쓰는 것 같아요. (찾아보기: 'kebab case', 'snake case', 'camel case')

아래는 작성해주신 코드를 케밥 케이스로 변경한 예시이고 파일명이 꼭 이래야한다는 절대적인 정답은 아닙니다!
```suggestion
    <link href="linkbrary-style.css" rel="stylesheet" />
```

파일명까지 포함해서 의견 드리자면, 
css 파일 자체가 스타일링을 한다는 것을 의미하기 때문에 'style' 이라는 단어가 파일명에 굳이 포함되지 않아도 될 것 같아요.
작성하신 것처럼 서비스의 명칭을 사용하여 `linkbrary.css` 로 작성하거나 홈페이지(index.html)를 의미하는 `index.css`, `home.css` 등으로도 작성해볼 수 있을 것 같네요 😀